examples/abbas_work_area/contact.py

```diff
@@ -0,0 +1,56 @@
+import file_manager
+
+"""
+This py file is the main file of this program, takes user inputs and sends them to the 
+file manager for processing.
+"""
+
+
+def main():
+    running = True  # stores the state of the application whether it's running or not
+    file_manager.if_header_exists()  # Calls the header function which creates a header for the csv file
```

Comment:
From the inline comment, we understand how this function is supposed to work. Just a suggestion to name the function like "write_header_if_it_not_exist"